### ISSUE TITLE
[css-text-3] Conformant use of text-transform

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#8279]
Currently we have the following note:
```
	Note: The 'text-transform' property only affects the presentation layer;
	correct casing for semantic purposes is expected to be represented
	in the source document.
```

I seem to have a stray file from 2020 on my disk which instead applies this expectation as conformance criteria:
```
	Advisement: Authors must not rely on 'text-transform' for semantic purposes;
	rather the correct casing and semantics should be encoded
	in the source document text and markup.
```

Filing this issue to ask whether we want to change it over or not.

**Comment by frivoal** [comment]
I am generally unconvinced of that conformance requirements on authors typically accomplish much (unless placed in a spec to which author conformance is sought and verified, like WCAG), but I am supportive of the idea expressed in this requirement.

I guess that makes me neutral on adopting this change.

**Comment by AutoSponge** [comment]
APA feels that the second choice of wording makes it more clear this the author's responsibility. Does changing from a "Note" to an "Advisement" change anything as far as spec language?

**Comment by frivoal** [comment]
A note is informative, an advisement is a (highlighted) normative part of the specification. If we want to use the second phrasing, it needs not to be merely an informative note, since it uses a RFC2119 verb ("must not").

So I guess that's what we'll do.

**Comment by frivoal** [comment]
We just resolved to republish text-3 as a CR, but I think it would be good to fold this in first. This seems good and non controversial, so maybe we can resolve it asynchronously?

Also, this is a requirement on authors, not on implementers, so it would not need tests.

cc: @astearns @atanassov

**Comment by astearns** [comment]
Since we have an APA request for promoting the text from Note to Advisement, we should likely make that change.

The CSSWG will automatically accept this resolution one week from now if no objections are raised here. Anyone can add an emoji to this comment to express support. If you do not support this resolution, please add a new comment.

Proposed Resolution: Make the authoring advice for `text-transform` normative

**Comment by astearns** [comment]
RESOLVED: Make the authoring advice for text-transform normative
